indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -301,4 +265,111 @@
             }
         }).isEverythingSelected;
     };
+
+    function _fetchSourceCategory(categoryId) {
+        if (!_categories[categoryId]) {
+            _categories[categoryId] = categoryId;
+            $.ajax({
+                url: build_url(Indico.Urls.Categories.info, {category_id: categoryId}),
+                dataType: 'json',
+                error: handleAjaxError,
+                success: function(data) {
+                    if (data) {
+                        _categories[categoryId] = data;
+                    }
+                }
+            });
+        }
+    }
+
+    function _moveCategories(ids, source, endpoint) {
+        var sourceId = _.isObject(source) ? source.category.id : source;
+        var data = {category_id: ids};
+
+        $('<div>').categorynavigator({
+            category: source,
+            confirmation: true,
+            openInDialog: true,
+            actionButtonText: $T.gettext("Move here"),
+            dialogTitle: $T.ngettext("Move category", "Move categories", ids.length),
+            dialogSubtitle: $T.ngettext("Select new category parent for the category",
+                                        "Select new category parent for {0} selected categories".format(ids.length),
```

Comment:
Doesn't matter, you can still call format on it - it simply won't do anything. But you cannot have dynamic parts inside `_()` (it breaks i18n)